src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -83,4 +83,12 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+    
+    default String getPrintableString(Printable... printables) {
```

Comment:
preferably don't use default access modifier. 
secondly, you can provide a header comment for this method, as its behavior is different from the one in the `Printable` interface